app/views/application_records/show.haml

```diff
@@ -1,15 +1,16 @@
-%h1 Application Record
+%h1.h2="#{@record.position.name} Application Record"
```

Comment:
What's with all the h-tags that have a CSS class named after another, smaller h-tag?